CONTRIBUTING.md

```diff
@@ -323,8 +323,8 @@ Before you can start, you need a GitHub account, here are a few suggestions:
     change the code by default.
 
 - Code specifics
-  - All significant new classes, methods and functions have Doxygen-style
-    documentation in source comments.
+  - All new public, protected, and private classes, methods, data members, and
```

Comment:
Sorry for the delay in this comment.  I keep thinking back to @pazner's point about something like
```
FiniteElementSpace fespace; // this is the finite element space
```
From what I understand, the current approach is that this rule of data members having comments will not be so rigorously applied as to enforce the above situation.  If that is correct, I'm personally leaning towards what I believe the prior iteration was, where there were "required" comments vs "recommended" comments... and having the data member comments be "recommended" when there is more than one finite element space (example stolen from someone else at the meeting) or the name is otherwise ambiguous.

That all said, I'm not against the current approach... so feel free to merge away.